randovania/data/json_data/prime2.txt

```diff
@@ -7819,35 +7819,66 @@ Asset id: 3468345533
   > Portal to Undertemple
       All of the following:
           Morph Ball
+          Before Hydrochamber Storage Item or After Alpha Blogg
           Any of the following:
-              All of the following:
-                  Spider Ball
-                  Any of the following:
-                      Before Hydrochamber Storage Item
-                      All of the following:
-                          After Hydrochamber Storage Item
-                          Gravity Boost or Movement (Intermediate)
+              Spider Ball
               Morph Ball Bomb and Space Jump Boots and Gravity Boost and Bomb Space Jump (Intermediate)
   > Door to Hydrodynamo Shaft
       Any of the following:
-          Morph Ball Bomb and Bomb Space Jump (Hypermode) and Air Underwater (Hypermode) and Use Screw Attack (Space Jump)
+          All of the following:
+              Morph Ball Bomb and Bomb Space Jump (Hypermode) and Air Underwater (Hypermode) and Use Screw Attack (Space Jump)
+              Before Hydrochamber Storage Item or After Alpha Blogg
           All of the following:
               Gravity Boost
               Any of the following:
                   All of the following:
-                      After Hydrochamber Storage Item
+                      After Alpha Blogg
                       Any of the following:
                           Space Jump Boots
                           Morph Ball and Roll Jump (Expert)
                   Before Hydrochamber Storage Item and Standable Terrain (Intermediate)
+                  All of the following:
+                      Morph Ball and Morph Ball Bomb and After Hydrochamber Storage Item
+                      Any of the following:
+                          Bomb Space Jump (Hypermode)
+                          Space Jump Boots and Bomb Space Jump (Intermediate)
+  > Event - Alpha Blogg
+      All of the following:
+          After Hydrochamber Storage Item
+          Any of the following:
+              All of the following:
+                  Gravity Boost or Combat (Advanced)
```

Comment:
On no tricks, this expects Gravity Suit and one of the ammo alternatives?

Reply:
Yes.

Reply:
non-ideal, but we can live with it